### PR TITLE
docs: add doc for opentelemetry logs

### PR DESCRIPTION
**Pull request by paomian**
## What's Changed in this PR

* add doc for opentelemetry logs

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/user-guide/ingest-data/for-observerbility/opentelemetry.md [hunk 1]
```diff
@@ -88,6 +88,90 @@ The example codes above may be outdated according to OpenTelemetry. We recommend
...
+
+You can find executable demos on GitHub at the links: [Go](https://github.com/GreptimeCloudStarters/quick-start-go), [Java](https://github.com/GreptimeCloudStarters/quick-start-java), [Python](https://github.com/GreptimeCloudStarters/quick-start-python), and [Node.js](https://github.com/GreptimeCloudStarters/quick-start-node-js).
+
+:::tip NOTE
+The example codes above may be outdated according to OpenTelemetry. We recommend that you refer to the official OpenTelemetry documentation for the most up-to-date information.
+:::
+
+For more information on the example code, please refer to the official documentation for your preferred programming language.
+
 ## Data Model
```
Comment (nicecui) [inline docs/user-guide/ingest-data/for-observerbility/opentelemetry.md:175]:
```suggestion
### Data Model
```
please move this section under the `## Metrics`

**Review by nicecui — COMMENTED** [review]
This PR needs further improvement:

1. Refine the outline of `opentelemetry.md`
2. Remove the origin `/docs/docs/db-cloud-shared/clients/otlp-integration.md`.
3. Add data model explanation for the logs
4. Modify the logging code that ensure it is executable.
5. Polish the English

**Review by sunng87 — APPROVED** [review]

File: i18n/zh/docusaurus-plugin-content-docs/current/db-cloud-shared/clients/otlp-logs-integration.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+GreptimeDB 是一个可观测性后端，能够通过 [OTLP/HTTP](https://opentelemetry.io/docs/specs/otlp/#otlphttp) 协议原生地消费 OpenTelemetry 日志。
+
+#### API
+
+要通过 OpenTelemetry SDK 库将 OpenTelemetry 日志发送到 GreptimeDB，请使用以下信息：
+
+* **URL:** `https://<host>/v1/otlp/v1/logs`
+* **Headers:**
+  * `X-Greptime-DB-Name`: `<dbname>`
+  * `Authorization`: `Basic` 认证，这是一个 Base64 编码的 `<username>:<password>` 字符串。更多信息，请参考 [鉴权](https://docs.greptime.com/zh/user-guide/deployments/authentication/static/) 和 [HTTP API](https://docs.greptime.com/user-guide/protocols/http#authentication)。
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/db-cloud-shared/clients/otlp-logs-integration.md:10]:
```suggestion
  * `Authorization`: `Basic` 认证，这是一个 Base64 编码的 `<username>:<password>` 字符串。更多信息，请参考 [鉴权](https://docs.greptime.cn/user-guide/deployments/authentication/static/) 和 [HTTP API](https://docs.greptime.cn/user-guide/protocols/http#authentication)。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/db-cloud-shared/clients/otlp-metrics-integration.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ GreptimeDB 通过原生支持 [OTLP/HTTP](https://opentelemetry.io/docs/specs/ot
 * URL: `https://<host>/v1/otlp/v1/metrics`
 * Headers:
   * `X-Greptime-DB-Name`: `<dbname>`
-* `Authorization`: `Basic` 认证，是 `<username>:<password>` 的 Base64 编码字符串。更多信息请参考 [鉴权](https://docs.greptime.cn/user-guide/deployments/authentication) 和 [HTTP API](https://docs.greptime.cn/user-guide/protocols/http#authentication)。
+* `Authorization`: `Basic` 认证，是 `<username>:<password>` 的 Base64 编码字符串。更多信息请参考 [鉴权](https://docs.greptime.com/zh/user-guide/deployments/authentication/static/) 和 [HTTP API](https://docs.greptime.cn/user-guide/protocols/http#authentication)。
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/db-cloud-shared/clients/otlp-metrics-integration.md:10]:
```suggestion
* `Authorization`: `Basic` 认证，是 `<username>:<password>` 的 Base64 编码字符串。更多信息请参考 [鉴权](https://docs.greptime.cn/user-guide/deployments/authentication/static/) 和 [HTTP API](https://docs.greptime.cn/user-guide/protocols/http#authentication)。
```

**Review by nicecui — APPROVED** [review]

**Pull request merged**